src/snackBar/Main.java

```diff
@@ -0,0 +1,72 @@
+package snackBar;
+
+public class Main
+{
+  public static void main(String[] args)
+  {
+    // instantiate customers
+    Customer c1 = new Customer("Jane", 45.25);
+    Customer c2 = new Customer("Bob", 33.14);
+
+    // instantiate vending machines
+    VendingMachine food = new VendingMachine("Food");
+    VendingMachine drink = new VendingMachine("Drink");
+    VendingMachine office = new VendingMachine("Office");
+
+    // instantiate snacks
+    // in Food
+    Snack snack1 = new Snack("Chips", 36, 1.75, food.getId());
+    Snack snack2 = new Snack("Chocolate Bar", 36, 1.00, food.getId());
+    Snack snack3 = new Snack("Pretzel", 30, 2.00, food.getId());
+
+    // in Drink
+    Snack snack4 = new Snack("Soda", 24, 2.50, drink.getId());
+    Snack snack5 = new Snack("Water", 20, 2.75, drink.getId());
+
+    // processing
+    // 1 - Customer 1 buys 3 of snack 4. Print Customer 1 Cash on hand. Print quantity of snack 4.
+    c1.buySnack(snack4.getTotalCost(4));
+    snack4.buySnack(4);
+    System.out.println("Customer 1 cash on hand: " + c1.getCash());
```

Comment:
As if 3 sodas weren't enough for Jane (she was supposed to buy 3 of snack 4), you sold her 4. Jane now needs a `visitDoctor` method because her blood sugar is running extremely high.